recipes/util-linux/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "util-linux" %}
+{% set version = "2.30" %}
+{% set md5 = "90e0722c82767c98861f608f9845345b" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://www.kernel.org/pub/linux/utils/util-linux/v2.30/util-linux-2.30.tar.gz
+    md5: {{ md5 }}
+
+build:
+    number: 0
+    # Skipping osx as this is really a linux only library. (util-linux in the name is a give-away..)
+    skip: True  # [win or osx]
+    
+requirements:
+    build:
+        - gcc
+        - automake
+        - autoconf
+        - bison
+        - pkg-config
+        - libtool
+        - gettext
+    
+    run:
+        - libgcc
```

Comment:
@jakirkham for the R packages we are already using gcc and libgcc in combination. With the upcoming GCC package it might be good to leave this and annotate gcc explicitly?